php/coresky/main/rewrite.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
Can this file be generated during the bootstrap step ?

Reply:
The same answer like for file: **php/coresky/main/gate.php**. Develoment tool generate this files (from web-interface data) on the develoment stage only.

Reply:
> Can this file be generated during the bootstrap step ?

No, this is an application code.